XpringKit/DefaultXpringClient.swift

```diff
@@ -1,5 +1,22 @@
 /// An interface into the Xpring Platform.
 public class DefaultXpringClient {
+  /// A network client that will make and receive requests.
+  private let networkClient: NetworkClient
+
+  /// Initialize a new XpringClient.
+  ///
+  /// - Parameter grpcURL: A url for a remote gRPC service which will handle network requests.
+  public convenience init(grpcURL: String) {
+    let networkClient = Rpc_V1_XRPLedgerAPIServiceServiceClient(address: grpcURL, secure: false)
```

Comment:
Why `secure: false`?

Reply:
This is whether or not to use SSL when connecting across gRPC. The remote node doesn't have an SSL certificate, so we use plaintext. 

We thought about how big of a problem this was and concluded:
- Anything that we're injecting to the node (i.e. transactions) is being sent in a broadcast context as it's being applied to the ledger
- Anything we are reading from the node is already publicly readable

So we don't think this is high on the priority list of features to support.